services/visual-studio-app-center/visual-studio-app-center-builds.service.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const {
+  BaseVisualStudioAppCenterService,
+  keywords,
+  documentation,
+} = require('./visual-studio-app-center-base')
+const { NotFound } = require('..')
+
+const schema = Joi.array().items({
+  result: isBuildStatus.required(),
+})
+
+module.exports = class VisualStudioAppCenterBuilds extends BaseVisualStudioAppCenterService {
+  static get category() {
+    return 'build'
+  }
+
+  static get route() {
+    return {
+      base: 'vsac/builds',
```

Comment:
Although it admittedly makes for a longer url, I think it'd be better to use the full `visual-studio-app-center` here like we do for the Visual Studio Marketplace badges: https://github.com/badges/shields/blob/master/services/visual-studio-marketplace/visual-studio-marketplace-downloads.service.js#L25

Reply:
Done in 10bf8ea